features/Makefile

```diff
@@ -11,7 +11,9 @@ ginkgo:
 	go get github.com/onsi/ginkgo/ginkgo
 
 functests: ginkgo
-	GOFLAGS=-mod=vendor ginkgo functests -- -junit /tmp/artifacts/unit_report.xml
+	FOCUS=$$(echo $(FEATURES) | tr ' ' '|') && \
+	echo "Focusing on $$FOCUS" && \
+	GOFLAGS=-mod=vendor ginkgo functests --focus=$$FOCUS -- -junit /tmp/artifacts/unit_report.xml
```

Comment:
when we use --focus, does the exit code of the test still reflect the test results when items are skipped.

The only reason I'm asking this is because when i use the 'F' on a 'Context' or 'Describe' to enable focus in the test suite... like 'FContext' to execute just the tests under that block... I get a non-zero exit code regardless if the tests passed or not. 

I just want to double check that the --focus cli arg doesn't have similar behavior.

Reply:
Tried it locally, I think it only applies to hard coded focused tests:
Detected Programmatic Focus - setting exit status to 197